docs.markdown

```diff
@@ -228,6 +262,28 @@ To have access to Kontist API endpoints that require strong customer authenticat
 
 We provide a simplified push notification MFA flow for users who have installed the Kontist Application and paired their device in it.
 
+<div class="mermaid">
+sequenceDiagram
+    participant Your App
+    participant Kontist API
+    participant User
```

Comment:
Shouldn't it be "Kontist Application" here instead of user? 🤔

Reply:
True, makes more sense. Adjusted that.